content/en/agent/guide/datadog-disaster-recovery.md

```diff
@@ -0,0 +1,284 @@
+{{% /collapse-content %}}
+
+
+{{% collapse-content title=" 7. Set up Resources syncing and scheduler" level="h5" %}}
+Datadog provides a tool called [Datadog sync-cli][3] to copy your dashboards, monitors and other configurations from your primary organization to your secondary organization. You can determine the frequency and timing of syncing based on your business requirements. Regular syncing is essential to ensure that your secondary organization is up-to-date in the event of a disaster. We recommend performing this operation on a daily basis. For information on setting up and running the backup process, see the [datadog-sync-cli README][5]. 
+
+Sync-cli is primarily intended for unidirectional copying and updating resources from your primary org to your secondary org. Resources copied to the secondary organization can be edited, but any new syncing will override changes that differ from the source in the primary organization.
+
+Each item can be added to the sync scope using the sync-cli configuration available in the documentation. Here's an example of a configuration file for syncing specific dashboards and monitors using name and tag filtering from an `EU` site to a `US5` site.
+
+```shell 
+destination_api_url="https://api.us5.datadoghq.com"
+destination_api_key="<US5_API_KEY>"
+destination_app_key="<US5_APP_KEY>"
+source_api_key="<EU_API_KEY>"
+source_app_key="<EU_APP_KEY>"
+source_api_url="https://api.datadoghq.eu"
+filter=["Type=Dashboards;Name=title","Type=Monitors;Name=tags;Value=sync:true"]
+
+# Make sure to increase the retry timeout to cope with the rate limit
+http_client_retry_timeout=600
+```
+
+Here's an example of a Sync-cli commands for syncing log configurations:
+
+```shell
+datadog-sync import –config config –resources users,roles,logs_pipelines,logs_pipelines_order,logs_indexes,logs_indexes_order,logs_metrics,logs_restriction_queries
+
+# remember to set the –cleanup=Force option
+datadog-sync sync –config config –resources users,roles,logs_pipelines,logs_pipelines_order,logs_indexes,logs_indexes_order,logs_metrics,logs_restriction_queries –cleanup=Force
+```
+
+<div class="alert alert-warning"> <strong>Sync-cli Limitation </strong><br><br>
+
+**Log Standard Attributes:** Sync-cli is regularly being updated with new resources. Currently, syncing Log standard attributes is not supported for private beta. If you use standard attributes with your log pipelines and are remapping your logs, attributes are a dependency that you need to manually re-configure in your secondary org. You can refer to the Datadog [standard attribute documentation][6] for support.
+</div>
+{{% /collapse-content %}}
+
+
+{{% collapse-content title=" 8. Verify availability at the secondary site" level="h5" %}}
+Verify that your secondary org is accessible and that your Dashboards and Monitors are copied from your primary org to your secondary org.
```

Comment:
How would someone verify this? What are they looking for? If they're lost, how can you help them?

Additionally, this is so short, I don't think it's worth a whole click to open it in a drawer 😛